pkgs/development/tools/faas-cli/default.nix

```diff
@@ -9,15 +9,15 @@ in
 buildGoModule rec {
   pname = "faas-cli";
   # When updating version change rev.
-  version = "0.13.13";
-  rev = "72816d486cf76c3089b915dfb0b66b85cf096634";
+  version = "0.13.15";
+  rev = "b562392b12a78a11bcff9c6fca5a47146ea2ba0a";
   platform = faasPlatform stdenv.targetPlatform;
```

Comment:
Doesn't this belong into meta?

Reply:
We already discussed the need for this [here](https://github.com/NixOS/nixpkgs/pull/137102#discussion_r739041577). If it is considered better practice to put this in meta shall move move it.

Reply:
Sorry, I am not seeing the discussion in the link (it is related to the `rev`, not `platform`).

It is somewhere else :thinking: ?

Reply:
You are right, my mistake. I was confused and thought this was about the `rev` again.

The `platform` value is used in the linker flags. The `faasPlatform` function maps the cpu name to the format that is expected by faas-cli.

https://github.com/NixOS/nixpkgs/blob/255cebd8bc1abe11362751f96312c6e63e3e93c8/pkgs/development/tools/faas-cli/default.nix#L3-L7

Reply:
Ok, this I understood looking at the code. However do you need to expose it as a attribute? Couldn't you leave it on `let in` instead?

Reply:
I think if we inline it in line 33 there is no potential for confusion.